eventbroadcaster/eventbroadcaster.go

```diff
@@ -18,3 +18,13 @@ func EventBroadcaster() record.EventBroadcaster {
 
 	return eventBroadcaster
 }
+
+func DisruptionCronEventBroadcaster() record.EventBroadcaster {
+	correlator := record.CorrelatorOptions{
```

Comment:
This broadcaster is dedicated to the disruption cron reconciler, because the broadcaster drops events. I'm not confident of this configuration, but it works. For what I understand here, we allow 1000 events to be sent per seconds before the aggregation, but not sure at all.

Reply:
From my observation It seems the aggregator is dropping event even if its message is different when it reach the limit. So if we set 2 events per 300 seconds, the third event emitted with a same key  will be dropped by the aggregatore and we will not have the last state of the updated disruption cron. Only the first will be broadcasted to the notifiers. Another behavior I have noticed is, if a second event is emitted with the same message and the same key linked to the same object, it will be dropped even if it has not reach the limit of events. So by adding the hash of the spec inside the message we are only taking in account the modifications of the spec, because the message will of the event will be the same as the previous one. 
 
The 1000 events per seconds is really huge and I used this configuration for my test cases. I agree we should reduce to something more realistic. I propose to set the max events to 10 during a period of 300 seconds. It's completely arbitrary, but it will allow the user to edit a disruption cron an enough amount of times. WDYT?